### PR TITLE
Sprints/lifesentence

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Solution/Solution/Entities/Account/Entity.xml [hunk 1]
```diff
@@ -468,6 +503,41 @@
...
+          <AutoNumberFormat></AutoNumberFormat>
+          <IsSearchable>1</IsSearchable>
+          <IsFilterable>0</IsFilterable>
+          <IsRetrievable>1</IsRetrievable>
+          <IsLocalizable>0</IsLocalizable>
+          <Format>text</Format>
+          <MaxLength>160</MaxLength>
+          <Length>320</Length>
+          <displaynames>
+            <displayname description="Nom du compte" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Account/Entity.xml:533]: @JonathanThomassin @EvoPheonix translation error here.

File: src/Solution/Solution/Entities/Contact/Entity.xml [hunk 2]
```diff
@@ -1963,9 +2276,11 @@
           </optionset>
           <displaynames>
             <displayname description="Require Privileged Correspondence" languagecode="1033" />
+            <displayname description="Require Privileged Correspondence" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/Entity.xml:2279]: missing translation here @JonathanThomassin @EvoPheonix

File: src/Solution/Solution/Entities/Contact/FormXml/main/{7c2799bf-9232-ec11-b6e6-000d3a09f410}.xml [hunk 3]
```diff
@@ -141,6 +141,7 @@
                       <cell id="{ca355352-067e-5ed1-0779-de0b32a0015c}" showlabel="true" locklevel="0" labelid="{55892634-30ab-4fee-a7df-afece55009ed}">
                         <labels>
                           <label description="Multiple Complaint Strategy" languagecode="1033" />
+                          <label description="Multiple Complaint Strategy" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{7c2799bf-9232-ec11-b6e6-000d3a09f410}.xml:144]: @EvoPheonix @JonathanThomassin missing translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml [hunk 4]
```diff
@@ -120,6 +121,7 @@
                       <cell id="{7284da39-112d-44f2-be81-93c1ae483258}" showlabel="true" locklevel="0">
                         <labels>
                           <label description="Preferred Language of Communication" languagecode="1033" />
+                          <label description="Preferred Language" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml:124]: @EvoPheonix @JonathanThomassin missing translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml [hunk 5]
```diff
@@ -128,6 +130,7 @@
                       <cell id="{8d21ec42-5707-44ba-bab8-7521b3b7ec0c}" showlabel="true" locklevel="0">
                         <labels>
                           <label description="Require Privileged Correspondence" languagecode="1033" />
+                          <label description="Require Privileged Correspondence" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml:133]: @EvoPheonix @JonathanThomassin missing translation

File: src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml [hunk 6]
```diff
@@ -324,6 +328,7 @@
             <cell id="{e6c1e3e8-e555-461d-8223-e8e6fa589856}" showlabel="true" locklevel="0">
               <labels>
                 <label description="Duplicate Detection Result" languagecode="1033" />
+                <label description="Duplicate Detection Result" languagecode="1036" />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/Contact/FormXml/main/{e04a2da4-a478-4dfe-a3e9-3f83b55ef1a4}.xml:331]: @EvoPheonix @JonathanThomassin missing translation

File: src/Solution/Solution/Entities/ActivityPointer/RibbonDiff.xml [hunk 7]
```diff
@@ -4,235 +4,11 @@
...
-      <Actions>
-        <JavaScriptFunction FunctionName="XrmCore.Commands.Disassociate.gridDisassociateLegacy" Library="$webresource:Main_system_library.js">
-          <CrmParameter Value="SelectedEntityTypeCode" />
-          <CrmParameter Value="SelectedControl" />
-          <CrmParameter Value="FirstSelectedItemId" />
-        </JavaScriptFunction>
-      </Actions>
-    </CommandDefinition>
-  </CommandDefinitions>
+  <CommandDefinitions />
```
Comment (pierluc-ssoucy) [inline src/Solution/Solution/Entities/ActivityPointer/RibbonDiff.xml:7]: I am guessing this was all removed when we refreshed from baseline? @mitchkirady

Comment (mitchkirady) [reply src/Solution/Solution/Entities/ActivityPointer/RibbonDiff.xml:7]: @pierluc-ssoucy - it was reset when i fixed the "new activity" drop-down issue on the activities subgrid.

**Pull request merged**